src/Command/CreateClassCacheCommand.php

```diff
@@ -57,7 +57,9 @@ public function __construct(string $cacheDir, bool $debug)
 
     public function configure(): void
     {
-        $this->setDescription('Generate the classes.php files');
+        $this->setDescription('Generate the classes.php files')
+            ->setName(static::$defaultName)
```

Comment:
Please, add a comment explaining why is this call required ([ref](https://github.com/sonata-project/SonataBlockBundle/blob/f6bcbedcacb3314cd5b295f06eb34b006075df02/src/Command/DebugBlocksCommand.php#L41)).

Reply:
Or just a NEXT_MAJOR hint, so we can remove this when dropping support for Symfony 2

Reply:
fix it